src/main/java/org/mastodon/views/trackscheme/display/TrackSchemePanel.java

```diff
@@ -197,17 +198,18 @@ public TrackSchemePanel(
 			final TrackSchemeGraph< ?, ? > graph,
 			final HighlightModel< TrackSchemeVertex, TrackSchemeEdge > highlight,
 			final FocusModel< TrackSchemeVertex, TrackSchemeEdge > focus,
-			final TimepointModel timepoint,
+			final TimepointModel navigationTimepointModel,
+			final FadedModel< TrackSchemeVertex, TrackSchemeEdge > fadedModel,
 			final SelectionModel< TrackSchemeVertex, TrackSchemeEdge > selection,
 			final RootsModel< TrackSchemeVertex > rootsModel,
 			final NavigationHandler< TrackSchemeVertex, TrackSchemeEdge > navigation,
-			final TrackSchemeOptions optional )
+			final TrackSchemeOptions trackSchemeOptions )
```

Comment:
"optional" is not a good name. I agree. But the class is TrackSchemePanel. Please name the variable simply "options".

Reply:
There is already a local variable in this method called "options". I followed your advise to rename the name of the variable in the method body to options, but I had to rename the existing variable "options" to "optionValues" to make this possible.